### PR TITLE
graduation

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by saarques** [comment]
1. Checkbox incomplete--(Fill the form if you haven't already)
2. Files modified--2(Maintain the original hyphens at header and footers)
3. Profile picture--(Not Okay)--(Picture should be in 1:1, yours is not square)

**Comment by sakshamtaneja21** [comment]
@saarques  what dose this means 
Maintain the original hyphens at header and footers)

**Comment by saarques** [comment]
> @saarques what dose this means
> Maintain the original hyphens at header and footers)

I meant that the original markdown file should be the same as given in the original README file.

You should only change your name, institution name, picture name, quote and github username.

Anything else should be like it originally is.

I meant these hyphens in the header and footer. It may not be of much importance but will help in making clear that you followed the instructions. :)

--- # You need not remove this.
name: FULLNAME-OR-NICKNAME # No longer than 28 characters
institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
github_user: YOUR-GITHUB-USERNAME
---

**Comment by sakshamtaneja21** [comment]
please check is it correct or not

**Comment by saarques** [comment]
> please check is it correct or not

Nope, I meant --- not ```

File: _data/sakshamtaneja21/sakshamtaneja21.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+```
```
Comment (saarques) [inline _data/sakshamtaneja21/sakshamtaneja21.md:1]: Use --- here, or copy paste from the original!

**Comment by sakshamtaneja21** [comment]
thanks for your help 
please check now

**Comment by sakshamtaneja21** [comment]
is it good to merge or not

**Comment by mitulsavani** [comment]
Hi @chandrikadeb7 thanks for your contribution on reviewing the PR, please refer to this comment https://github.com/education/graduation/issues/805#issuecomment-633178681 to go over some of the practices and problem that we have observed.

**Comment by chandrikadeb7** [comment]
> Hi @chandrikadeb7 thanks for your contribution on reviewing the PR, please refer to this comment [#805 (comment)](https://github.com/education/graduation/issues/805#issuecomment-633178681) to go over some of the practices and problem that we have observed.

Okay. Thanks.

**Comment by saarques** [comment]
> is it good to merge or not

It's all good, sit back and enjoy! :)

**Comment by sakshamtaneja21** [comment]
thanks, @juanpflores  
I have filled the form earlier also but now have filled it again

**Pull request merged**